var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -59,8 +59,8 @@ class NetlibLapack(Package):
     variant('xblas', default=False,
             description='Builds extended precision routines using XBLAS')
 
-    patch('ibm-xl.patch', when='@3.7: %xl')
-    patch('ibm-xl.patch', when='@3.7: %xl_r')
+    patch('ibm-xl.patch', when='@3.7:')
```

Comment:
Does it break the build with GCC or Clang+gfortran?

Reply:
@davydden This should not, as the patch applies to a portion of code that is only run when cmake detects that the Fortran compiler is an IBM XL or VisualAge comipler.

Reply:
@davydden This should not, as the patch applies to a portion of code that is only run when cmake detects that the Fortran compiler is an IBM XL or VisualAge compiler.  I did verify that it did not break the build with gcc.  However, the clang+gfortran build appears to be broken.  It may already have been broken on Power, but I'll poke at it for a bit.

Reply:
You can try building without this PR, and if’s broken, then most likely it was so already

Reply:
I'm not sure why it was breaking before, but applying my patch over a clean Spack installation and its working now.